docs/QUnit/test.each.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: QUnit.test.each()
+excerpt: Add a parameterized test to run.
+categories:
+  - main
+  - async
```

Comment:
I think we should omit it from the `async` category. That's something we do only for `QUnit.test()` for legacy reasons as `asyncTest` used to be there. The other variants aren't under there.

Reply:
Makes sense. Removed.